pymc3/distributions/timeseries.py

```diff
@@ -464,6 +464,86 @@ def logp(self, x):
     def _distr_parameters_for_repr(self):
         return ["mu", "cov"]
 
+    def random(self, point=None, size=None):
+        """
+        Draw random values from MvGaussianRandomWalk.
+
+        Parameters
+        ----------
+        point: dict, optional
+            Dict of variable values on which random values are to be
+            conditioned (uses default point if not specified).
+        size: int, optional
+            Desired size of random sample (returns one sample if not
+            specified).
+
+        Returns
+        -------
+        array
+
+
+        Examples
+        -------
+        Create one sample from a 2-dimensional Gaussian random walk with 10 timesteps::
+
+            mu = np.array([1.0, 0.0])
+            cov = np.array([[1.0, 0.0], [0.0, 2.0]])
+            sample = MvGaussianRandomWalk(mu, cov, shape=(10, 2)).random(size=1)
+        """
+
+        param_attribute = getattr(
+            self.innov, "chol_cov" if self.innov._cov_type == "chol" else self.innov._cov_type
+        )
+        mu, param = distribution.draw_values(
+            [self.innov.mu, param_attribute], point=point, size=size
+        )
+        return distribution.generate_samples(
+            self._random,
+            size=size,
+            dist_shape=self.shape,
+            not_broadcast_kwargs={
+                "sample_shape": to_tuple(size),
+                "param": param,
+                "mu": mu,
+                "cov_type": self.innov._cov_type,
+            },
+        )
+
+    def _random(self, mu, param, size, sample_shape, cov_type):
```

Comment:
```suggestion
multivariate_samples = self.innov.random(point=point, size=size)
```
How about the use of MvNormal random method here?
Because if `cov` is a random variable, then drawing samples from it can have more than 2 dimensions and scipy.stats.multivariate_normal cannot handle a batch of multivariate normals.

Reply:
That sounds like a great idea. What is the shape of the output of `MvNormal`'s random array? I'm trying to think of a good way to select out the time axis to perform the cumulative sum.

Reply:
> What is the shape of the output of MvNormal's random array

If you pass (time_dims, n_events) shape to MvNormal, then output shape will be (sample_shape, time_dims, n_events).